static/js/portico/landing-page.js

```diff
@@ -95,19 +95,29 @@ const apps_events = function () {
         const $download_instructions = $(".download-instructions");
         const $third_party_apps = $("#third-party-apps");
         const $download_android_apk = $("#download-android-apk");
+        const $download_from_google_play_store = $(".download-from-google-play-store");
+        const $download_from_apple_app_store = $(".download-from-apple-app-store");
+        const $link = $(".link");
         const version_info = info[version];
 
         $(".info .platform").text(version_info.alt);
         $(".info .description").text(version_info.description);
         $(".info .link").attr("href", version_info.link);
+        $(".download-from-google-play-store").attr("href", version_info.link);
+        $(".download-from-apple-app-store").attr("href", version_info.link);
         $(".image img").attr("src", version_info.image);
         $download_instructions.find("a").attr("href", version_info.install_guide);
 
+        $link.toggle(version === "mac" ||
+                    version === "windows" ||
+                    version === "linux");
```

Comment:
This indentation is inconsistent.  I'd also extract a `$desktop_app` variable to avoid duplicating this conditional already present below.

Or probably even better, expanding `version_info` to have an `app_type` field that's either `mobile` or `desktop and doing `$desktop_download_link.toggle(version_info.app_type === "desktop")` here.

Reply:
> Or probably even better, expanding `version_info` to have an `app_type` field that's either `mobile` or `desktop and doing `$desktop_download_link.toggle(version_info.app_type === "desktop")` here.

yes this seems to be a better approach